sass/style.scss

```diff
@@ -940,3 +944,19 @@ div.figure img {
 .contact-modal #message-text {
   height: 120px;
 }
+
+/****************************************************************
+  * NAVBAR LARGE TABLET / SMALL DESKTOP SIZE TRANSITIONAL STUFF *
+ ****************************************************************/
+
+@media (max-width: 1265px) and (min-width: $breakpoint-md) {
+  .navbar.ftm .navbar-nav .nav-link {
+    font-size: 10.5px;
```

Comment:
can you change this (and line 960) to `rem` to be consistent with our other font-size specs? Also, I think we can get away with it being a bit bigger, at least testing it in Brave right now. I tried `0.8rem` and it worked

Reply:
Done!  I was testing widths all the way down to `992px` and `0.8rem` is a little bit too big for the `iPad Pro` type of screen.  `0.67rem` is about as big as we can go here

Reply:
sounds good thanks!